after/syntax/python.vim

```diff
@@ -30,3 +30,5 @@ if g:jedi#show_call_signatures > 0 && has('conceal')
         hi jediFat term=bold,underline cterm=bold,underline gui=bold,underline ctermbg=0 guibg=#555555
     endif
 endif
+
+hi jediUsage term=NONE cterm=NONE ctermfg=None guifg=None gui=NONE ctermbg=3 guibg=Yellow
```

Comment:
The color (ctermbg=3) does not look good for me.
What about using `hi jediUsage term=standout cterm=standout gui=standout` ?